dashboard-frontend/src/types/account.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiResponse } from './api';
+
+export interface AccountsApiResponse extends ApiResponse<AccountResource[]> {}
+export interface AccountApiResponse extends ApiResponse<AccountResource> {}
+
+export interface CreateAccountApiResponse
+  extends ApiResponse<AccountResource> {}
+
+export interface AccountResource {
+  id?: string;
+  name: string;
+  authType: string;
+  connectedAccount?: ConnectedAccountResource;
+}
+
+export interface ConnectedAccountResource {
```

Comment:
I think we can drop Resource from our naming now. This was something Eric implemented but I think it's a little wordy. Connected Account etc still makes sense to me